modules/ocf_desktop/files/xsession/pdf-open

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+[ "$#" -ne 1 ] && { echo 'Bad arguments, trying evince...'; exec evince "$@"; }
+[ -f "$1" ] || { echo 'File does not exist, trying evince...'; exec evince "$@"; }
+
+size=$(du "$1") || { echo 'Cannot get size, trying evince...'; exec evince "$@"; }
+size=$(cut -f1 <<< "$size")
+[ "$size" -lt 5120 ] || { echo 'Too large, trying evince...'; exec evince "$@"; }
+
+echo 'Rasterizing PDF...'
+tmp=$(mktemp --suffix='.pdf')
+
+if ! convert -density 250 "$1" "$tmp"; then
+    rm "$tmp"
+    echo 'Convert failed, trying evince...'
+    exec evince "$@"
+fi
+
+echo 'Done rasterizing, now overwriting unrasterized version...'
+exiftool -tagsfromfile "$@" -title "$tmp"
+mv "$tmp" "$@"
```

Comment:
i don't think we should ever overwrite peoples' PDFs just because they opened it in (what they thought was) evince

we don't know what they're going to do with it afterwards -- printing is just one possible thing they might do. maybe they're just opening it before posting on a website or something. i'd be pretty annoyed if opening "evince" resulted in my files being overwritten with a rasterized version